custom_components/tuya_local/devices/tuya_gas_sensor.yaml

```diff
@@ -0,0 +1,12 @@
+name: TUYA Gas Leak Detector
+products:
+  - name: TUYA Gas Leak Detector
+primary_entity:
+  entity: sensor
+  class: carbon_dioxide
+  name: Sensor Reading
+  dps:
+    - id: 5
+      type: float
```

Comment:
Is it really float?  Most Tuya devices are using integer, and then apply a scale factor to get fixed point.

Reply:
Need confirmation on this to proceed to merge